### PR TITLE
Query by issuer test.

### PR DESCRIPTION
**Pull request by roger3cev**

### CODE REVIEW

File: workflows/src/test/kotlin/com/r3/corda/lib/tokens/workflows/TokenSelectionTests.kt [hunk 1]
```diff
@@ -93,6 +102,37 @@ class TokenSelectionTests : MockNetworkTest(numberOfNodes = 4) {
         // TODO: Assert something...
     }
 
+    fun <T : TokenType> issuerCriteria(token: T, issuer: Party): QueryCriteria {
```
Comment (kasiastreich) [inline workflows/src/test/kotlin/com/r3/corda/lib/tokens/workflows/TokenSelectionTests.kt]: We already have issuer criteria helper function `tokenAmountWithIssuerCriteria` and ownedTokensByTokenIssuer

File: workflows/src/test/kotlin/com/r3/corda/lib/tokens/workflows/TokenSelectionTests.kt [hunk 2]
```diff
@@ -93,6 +102,37 @@ class TokenSelectionTests : MockNetworkTest(numberOfNodes = 4) {
...
+        val tokenClassCriteria = QueryCriteria.VaultCustomQueryCriteria(builder { PersistentFungibleToken::tokenClass.equal(token.tokenClass) })
+        val tokenIdentifierCriteria = QueryCriteria.VaultCustomQueryCriteria(builder { PersistentFungibleToken::tokenIdentifier.equal(token.tokenIdentifier) })
+        val issuerCriteria = QueryCriteria.VaultCustomQueryCriteria(builder { PersistentFungibleToken::issuer.equal(issuer) })
+        return tokenClassCriteria.and(tokenIdentifierCriteria).and(issuerCriteria)
+    }
+
+    @Test
+    fun `select tokens by issuer`() {
```
Comment (kasiastreich) [inline workflows/src/test/kotlin/com/r3/corda/lib/tokens/workflows/TokenSelectionTests.kt]: We also... have tests ```com.r3.corda.lib.tokens.workflows.TokenQueryTests#query for sum of an owned token amount by issuer```

File: workflows/src/test/kotlin/com/r3/corda/lib/tokens/workflows/TokenSelectionTests.kt (hunk 1 above)
Comment (roger3cev) [reply workflows/src/test/kotlin/com/r3/corda/lib/tokens/workflows/TokenSelectionTests.kt]: whoops thanks using that now.....

**Comment by roger3cev** [comment]
over to you @kasiastreich 
thanks!

**Pull request merged**